wallet-connect-test-bench/front-end/package.json

```diff
@@ -20,6 +20,10 @@
         "react-dom": "^18.1.0",
         "react-switch": "^7.0.0"
     },
+    "resolutions": {
+        "@concordium/wallet-connectors": "../../deps/concordium-dapp-libraries/packages/wallet-connectors",
+        "@concordium/react-components": "../../deps/concordium-dapp-libraries/packages/react-components"
+    },
```

Comment:
Now that `@concordium/react-components` and `@concordium/wallet-connectors` are proper dependencies, we should be able to remove the following dependencies:

- `@concordium/browser-wallet-api-helpers`
- `@walletconnect/qrcode-modal`
- `@walletconnect/sign-client`

Reply:
Good point.

Reply:
Removed now.